packages/map-template/README.md

```diff
@@ -62,4 +62,5 @@ To have any Stencil component changes be reflected in this project, you need to
 |`venue`|`string`|The Venue to load from your MapsIndoors Solution. |
 |`locationId`|`string`|Set a MapsIndoors Location ID to show it on the map and its details in the sheet. |
 |`primaryColor`|`string`|The primary color to use throughout the app. |
-|`logo`|`string`|The logo to show during initial load. |
\ No newline at end of file
+|`logo`|`string`|The logo to show during initial load. |
+|`tileStyle`|`string`|Tile style name to change the interface of the map. |
```

Comment:
@markhougaard Can we get your input to the phrasing here?

Reply:
Maybe this?

```suggestion
|`tileStyle`|`string`|Name of Tile Style to display on the map. |
```